caddy/caddymain/run_test.go

```diff
@@ -15,8 +15,12 @@
 package caddymain
 
 import (
+	"reflect"
 	"runtime"
+	"strings"
 	"testing"
+
+	_ "github.com/mholt/caddy/caddyhttp"
```

Comment:
What is this import for?

Reply:
Let me check it, i believe it could be here from previous iterations:)

Reply:
Fixed this import:)